exokit.cpp

```diff
@@ -32,11 +32,11 @@ void callFunction(const char *funcname, const int argc, Local<Value> argv[]) {
 
   // call function, 'this' points to global object
   TryCatch try_catch(Isolate::GetCurrent());
-  Local<Value> result = jsfunc->Call(global, argc, argv);
+  Local<Value> result = Nan::Call(jsfunc, global, argc, argv).ToLocalChecked();
```

Comment:
Is it intentional to use `Nan::Call` here rather than doing `jsfunc->Call(JS_CONTEXT(), ...)`?

Reply:
I think it should be the same thing, but it would be good to be consistent.